packages/cycle-scripts-one-fits-all/template/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compileOnSave": false,
     "compilerOptions":
     {
-        "module": "ES6",
```

Comment:
I would leave this file unchanged, `es2015` e.g. is not a valid value (`ES2015` would be)